docs/preview/features/correlation.md

```diff
@@ -108,4 +108,64 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 }
 ```
 
+## Using secret store within Azure Functions
```

Comment:
Let's update the docs according to how it works now.

Reply:
Updated the suggessted approach and added a note to how you can use your own injected `ICorrelationInfoAccessor`.